docs/design/fabric-v2+/fpc-validation.puml

```diff
@@ -0,0 +1,166 @@
+/'
+    Copyright 2020 Intel Corporation
+    Copyright IBM Corp. All Rights Reserved.
+
+    SPDX-License-Identifier: Apache-2.0
+'/
+
+@startuml
+
+'- force wrapping of longer text
+'skinparam wrapWidth 400
+
+!pragma teoz true
+
+hide footbox
+title Foot Box removed
+
+title Fabric Private Chaincode - Transaction validation using FPC validation plugin
+
+actor "User Org1" as User1 order 9
+
+participant "Peer Org1" as ECC1 order 30
+participant "Peer Org2" as ECC2 order 60
+participant "Peer Org3" as ECC3 order 70
+
+entity Orderer order 100
+
+note right User1
+NOTE This validation flow only supports designated peer
+endorsement policy as specified for MVP. That is, this
+validation design will be replaced in the future with
+MSP-based validation.
+end note
+
+ref over User1, Orderer
+    install, approve, commit chaincode definition (ECC_MR)
+    see fpc-lifecycle-v2.puml
+end ref
+
+ref over User1, Orderer
+    register enclaves and key-dist
+    see fpc-registration.puml and fpc-key-dist.puml
+end ref
+
+group endorsement
+    activate User1
+
+    User1 -> ECC1 ++: send proposal
+    ref over ECC1
+        invoke chaincode enclave
+        see fpc-cc-invocation.puml
+    end ref
+    return proposal response
+
+    User1 -> ECC2 ++: send proposal
+    ref over ECC2
+        invoke chaincode enclave
+        see fpc-cc-invocation.puml
+    end ref
+    return
```

Comment:
Given that this is a "designated peer" scenario, it is misleading to have multiple endorsers.

```suggestion
```